src/test/java/org/zalando/problem/spring/web/advice/StatusConverterTest.java

```diff
@@ -49,9 +49,11 @@ public void shouldConvertHttpStatusToStatusType() {
     }
 
     @Test
-    public void shouldConvertUnsupportedHttpStatusToUnknownStatus() {
+    public void shouldSuccessfullyConvertNonStandardStatus() {
         StatusType statusType = unit.convert(INSUFFICIENT_SPACE_ON_RESOURCE);
-        assertThat(statusType, instanceOf(UnknownStatus.class));
+        assertThat(statusType.getStatusCode(), is(INSUFFICIENT_SPACE_ON_RESOURCE.value()));
```

Comment:
I wouldn't mind seeing concrete values, i.e. string literals, for verification. I know it would break if spring decides to chenge them, but I personally feel like tests are more readable that way.